libraries/APM_Control/AR_AttitudeControl.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include <AP_AHRS/AP_AHRS.h>
+#include <AP_Common/AP_Common.h>
+#include <AC_PID/AC_PID.h>
+#include <AC_PID/AC_P.h>
+
+// attitude control default definition
+#define AR_ATTCONTROL_STEER_ANG_P       1.00f
+#define AR_ATTCONTROL_STEER_RATE_P      1.00f
+#define AR_ATTCONTROL_STEER_RATE_I      0.50f
+#define AR_ATTCONTROL_STEER_RATE_IMAX   1.00f
+#define AR_ATTCONTROL_STEER_RATE_D      0.00f
+#define AR_ATTCONTROL_STEER_RATE_FILT   5.00f
+#define AR_ATTCONTROL_THR_SPEED_P       0.20f
+#define AR_ATTCONTROL_THR_SPEED_I       0.20f
+#define AR_ATTCONTROL_THR_SPEED_IMAX    1.00f
+#define AR_ATTCONTROL_THR_SPEED_D       0.00f
+#define AR_ATTCONTROL_THR_SPEED_FILT    5.00f
+#define AR_ATTCONTROL_DT                0.02f
+
+// throttle/speed control defaults
+#define AR_ATTCONTROL_THR_ACCEL_MAX     5.00f
+
+// minimum speed
```

Comment:
Well, it's SI units so normally we don't specify in the name in these cases I think... and it's a parameter default value and the parameter description has the units listed.. but I'll add the units in any case in the comments above.